compiler/crates/relay-compiler/tests/compile_relay_artifacts/fixtures/relay-resolver-with-args-fragment-spread.expected

```diff
@@ -0,0 +1,225 @@
+==================================== INPUT ====================================
+fragment relayResolverWithArgsFragmentSpread_PopStarNameResolverFragment_name on ClientType @argumentDefinitions(scale: {type: "Float!"}) {
+  name
+  profile_picture(scale: $scale)
+}
+
+query relayResolverWithArgsFragmentSpread_Query {
+  ...relayResolverWithArgsFragmentSpread_Fragment
+}
+
+fragment relayResolverWithArgsFragmentSpread_Fragment on Query {
+  pop_star {
+    pop_star_name(scale: $scale)
```

Comment:
The variable $scale is not defined on the `relayResolverWithArgsFragmentSpread_Query`. We should throw a validation error in this case.